specifications/providers/openstack/flavors_and_images.md

```diff
@@ -136,6 +140,10 @@ As defined by this specification we expect:
 }
 ```
 
+The Image OS fields (name, version and family) describes the operating system in use. `os_name`, `os_version` and `os_family` defines the full name, version and family of the operating system.
```

Comment:
Stylistic thing that's not documented 😄  One line of text to one line of code please, it makes diffing easier in future.  Tip I learned from technical documentation people.  Besides if we ever had to write a docs parser, it's all in the same format then.